unit_tests/ut_h_hierarchy_by_name.amlg

```diff
@@ -78,85 +38,88 @@
 	(assign (assoc
 		result
 			(call_entity "howso" "copy_subtrainee" (assoc
-				target_trainee "c_copy"
-				source_name_path (list "invalid")
+				target_path ["c_copy"]
+				target_id "uniqueid"
+				source_path ["invalid"]
 			))
 	))
 	(print "Can't copy - bad child: ")
 	(call assert_same (assoc
 		obs (get result (list 1 "detail"))
-		exp "Invalid name path specified."
+		exp "source: Invalid path specified."
 	))
 
 	(assign (assoc
 		result
 			(call_entity "howso" "copy_subtrainee" (assoc
-				target_name_path (list "invalid")
-				target_trainee "c_copy"
-				source_name_path (list "A" "c")
+				target_path ["invalid" "c_copy"]
+				target_id "uniqueid"
+				source_path ["A" "c"]
 			))
 	))
 	(print "Can't copy - bad parent: ")
 	(call assert_same (assoc
 		obs (get result (list 1 "detail"))
-		exp "Invalid name path specified."
+		exp "Invalid path specified."
 	))
 
 	(assign (assoc
 		result
 			(call_entity "howso" "copy_subtrainee" (assoc
-				target_trainee "A"
-				source_name_path (list "A" "c")
+				target_path ["A"]
+				target_id "uniqueid"
+				source_path ["A" "c"]
 			))
 	))
 	(print "Can't copy - duplicate child: ")
 	(call assert_same (assoc
 		obs (get result (list 1 "detail"))
-		exp "Specified target_trainee already exists."
+		exp "Specified trainee is already in the hierarchy."
```

Comment:
Maybe instead use `A trainee already exists in the hierarchy at path "...".` instead